os/android/drmhwctwo.h

```diff
@@ -239,6 +239,9 @@ class DrmHwcTwo : public hwc2_device_t {
   hwcomposer::GpuDevice device_;
   std::map<hwc2_display_t, HwcDisplay> displays_;
   std::map<HWC2::Callback, HwcCallback> callbacks_;
+
+ private:
+  static bool is_explicit_sync_enabled_;
```

Comment:
1. As I mentioned:
renderer_ need to be initialized in Compositor::Init(). If not, below call would caused fault address error.
renderer_->SetExplicitSync(ExplicitSyncEnabled);
So if don't touch Compositor::Init(), we should use static is_explicit_sync_enabled_ in GLRenderer and EGLOffScreenContext.

2. DrmHwcTwo and DrmHwcTwo::HwcDisplay both use is_explicit_sync_enabled_ in os/android/drmhwctwo.cpp. If not use static, there is compile issue mentioned above.